src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -14,497 +14,427 @@ limitations under the License.
-                id="git-revision-text-input"
-                labelText="Revision"
-                helperText="Branch name or commit ID"
-                placeholder="master"
-                name="gitRevision"
-                invalidText={noSpacesInvalidText}
-                invalid={this.state.gitRevision.invalid}
-                value={this.state.gitRevision.value}
-                onChange={this.onInputChange}
-              />
-            </FormGroup>
-
-            <FormGroup legendText="Image Resource (optional)">
-              <TextInput
-                id="image-name-text-input"
-                labelText="Name"
-                placeholder="docker-image"
-                name="imageName"
-                invalidText={resourceNameInvalidText}
-                invalid={this.state.imageName.invalid}
-                value={this.state.imageName.value}
-                onChange={this.onInputChange}
-              />
-              <TextInput
-                id="image-registry-text-input"
-                labelText="Registry"
-                placeholder="registryname"
-                name="imageRegistryName"
-                invalidText={noSpacesInvalidText}
-                invalid={this.state.imageRegistryName.invalid}
-                value={this.state.imageRegistryName.value}
-                onChange={this.onInputChange}
-              />
-              <TextInput
-                id="image-repo-text-input"
-                labelText="Repository"
-                placeholder="reponame"
-                name="imageRepoName"
-                invalidText={noSpacesInvalidText}
-                invalid={this.state.imageRepoName.invalid}
-                value={this.state.imageRepoName.value}
-                onChange={this.onInputChange}
-              />
-            </FormGroup>
-
-            <FormGroup legendText="Helm (optional)">
-              <Toggle
-                id="helm-pipeline-toggle"
-                data-testid="helm-pipeline-toggle"
-                labelText="Toggle On when using a Helm pipeline"
-                name="helmPipeline"
-                toggled={this.state.helmPipeline.value}
-                onToggle={this.onHelmToggleChange}
-              />
-              <TextInput
-                id="helm-secret-text-input"
-                labelText="Secret"
-                placeholder="helm-secret"
-                name="helmSecret"
-                invalidText={resourceNameInvalidText}
-                invalid={this.state.helmSecret.invalid}
-                value={this.state.helmSecret.value}
-                onChange={this.onInputChange}
-              />
-            </FormGroup>
-          </ModalBody>
-          <ModalFooter>
-            <Button kind="secondary" onClick={this.onClose}>
-              Cancel
-            </Button>
-            <Button kind="primary" type="submit">
-              Submit
-            </Button>
-          </ModalFooter>
-        </ComposedModal>
+            <TextInput
```

Comment:
Should this be a NumberInput? What's the unit, seconds?

Reply:
Weirdly enough, I think it's actually a string, because it gets parsed here https://golang.org/pkg/time/#ParseDuration (mentioned here: https://godoc.org/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1#PipelineRunSpec)

Reply:
Ah I see, so a user could enter a value like "10m" for 10 minutes for example. We might need to provide a little more guidance in the UI for this but let's leave it as-is for now and we can revisit once we start using it ourselves.